README.md

```diff
@@ -30,6 +30,6 @@
 - Use `_` between a job name and the [job template variables](https://docs.openstack.org/infra/jenkins-job-builder/definition.html#job-template)
 - Use standard capitalization rules, template variables can be an exception to this
 - Examples:
-  - `RPC-AIO_{series}-{context}-{ztrigger}`
-  - `RPC-AIO_master-swift-periodic`
-  - `Merge-Trigger-JJB`
\ No newline at end of file
+  - `RPC-AIO_{series}-{image}-{scenario}-{action}-{ztrigger}`
```

Comment:
I'd be happy to either:
- Not make this too complex, and use this formatting for only greenfields (make upgrade and leapfrog a different beast)
- Make this more complete, and extend this kind of scenario for upgrades/leapfrog by adding a "{from_version}", which would increase the from->to matrix, necessary for leapfrogs next month.